src/test/logical_query_plan/create_table_node_test.cpp

```diff
@@ -11,7 +11,7 @@ class CreateTableNodeTest : public ::testing::Test {
   void SetUp() override {
     column_definitions.emplace_back("a", DataType::Int, false);
     column_definitions.emplace_back("b", DataType::Float, true);
-    create_table_node = CreateTableNode::make("some_table", column_definitions);
+    create_table_node = CreateTableNode::make("some_table", column_definitions, false);
```

Comment:
@mrks I insist that having a bool parameter here is NOT readable.
(@d-justen ignore this comment)